content/about/contributing.md

```diff
@@ -30,7 +30,7 @@ If you are certain you have found a bug, just register a ticket with the relevan
 ## Help us help you
 
 You will likely use Cucumber combined with various web frameworks, many different dependencies and your own code.
-You can [run Features](/configuration/#running-features) in different ways, and many things can go wrong - and quite often it has nothing to do with Cucumber at all.
+You can [run Features](/running/) in different ways, and many things can go wrong - and quite often it has nothing to do with Cucumber at all.
```

Comment:
Change "run Features" -> "run features" (for consistency) or "run Cucumber" like in the menu

Reply:
I'm pretty sure I didn't touch "run Features". But I agree and fixing your suggestions is a good thing to do.

Reply:
Yes, I know. Thanks for fixing anyway :)